README.md

```diff
@@ -40,7 +44,7 @@ Notice that either way `ATLAS_DATA_PATH` will be created for you if the director
 Besides `ATLAS_DATA_PATH` you need to set some variables for the Postgres container. Use the settings below and replace
 `ATLAS_DATA_PATH` value to the directory you set up in the first step.
 
-In the `atlas-web-single-cell/docker` directory run the following:
+In the `atlas-web-bulk-cell/docker` directory run the following:
```

Comment:
The following bash code is not working for the new docker compose version anymore, please modify it to the latest command.